package.json

```diff
@@ -90,7 +90,10 @@
     "build:cds:ssr": "cross-env SPARTACUS_CDS=true ng run storefrontapp:server:production",
     "prerender:cds": "cross-env SPARTACUS_CDS=true ng run storefrontapp:prerender",
     "release:cds:with-changelog": "cd projects/cds && release-it && cd ../..",
-    "build:myaccount": "ng build my-account --prod",
+    "cleanup:myaccount:schematics": "rimraf \"feature-libs/my-account/schematics/**/*.js\" \"feature-libs/my-account/schematics/**/*.js.map\" \"feature-libs/my-account/schematics/**/*.d.ts\"",
```

Comment:
Can we move this command and the build one to my-account package.json and only reference it here (the build command, cleanup only there)?

Reply:
Done